src/aux-records/ServerConfig.ts

```diff
@@ -882,6 +882,35 @@ const telemetrySchema = z.object({
         .default({}),
 });
 
+const tigerBeetleSchema = z
+    .object({
+        _enabled: z
+            .boolean()
+            .optional()
+            .default(false)
+            .describe(
+                'Whether TigerBeetle is enabled, this allows for pre-defining and shipping default configuration.'
+            ),
```

Comment:
@LandonSiler We don't need a flag for if tiger beetle is enabled. We can simply allow the tigerBeetle key in the server config to be null/undefined to disable it.